src/api/exchange/ExchangeApiImpl.ts

```diff
@@ -58,6 +60,7 @@ const decodeAuctionElements = (bytes: string): AuctionElement[] => {
     result.push({
       user: '0x' + user,
       sellTokenBalance: new BN(sellTokenBalance, 16),
+      id: index++,
```

Comment:
My idea is to have it from the API, because later we might re-order the list based on different sorting factors and lose the initial index. Also, we might eventually add pagination

Reply:
valid point